docs/source/user_guide.rst

```diff
@@ -0,0 +1,606 @@
+.. _user-guide:
+
+User Guide
+#################
+
+This guide help you getting started with the Neo4j GenAI Python package,
+and explain how to configure it to meet your requirements.
+
+
+************
+Quick start
```

Comment:
Very pedantic but should we have Title Case? I do this elsewhere in the documentation. Happy with either as long as we are consistent

Reply:
You're right, seems to be the convention also for other neo4j docs. I've updated all titles.